dev-environment.yml

```diff
@@ -37,6 +37,7 @@ dependencies:
   - ipywidgets
   - jupyter_contrib_nbextensions
   - notebook
+  - pooch
```

Comment:
Recommend adding an EOL comment here like `- pooch  # for scikit-image` so we don't prune it by accident later.

Reply:
Fixed in 83ddf0a